src/promptflow/promptflow/_sdk/_submitter/test_submitter.py

```diff
@@ -396,6 +397,24 @@ def _get_generator_outputs(outputs):
 class TestSubmitterViaProxy(TestSubmitter):
     def __init__(self, flow: Flow, flow_context: FlowContext, client=None):
         super().__init__(flow, flow_context, client)
+        self._init_executor()
+
+    def _init_executor(self):
+        logger.info("Promptflow Proxy executor starts initializing...")
```

Comment:
if we init it here, we will also need to call `destory` on the executor proxy explicitly somewhere.

And please use names different from `Executor` for `ExecutorProxy`

Reply:
removed